File: Python Starter Code/bot.py
# bot.py
import os
import discord
import random
import re
import json


from dotenv import load_dotenv

load_dotenv()
TOKEN = os.getenv('DISCORD_TOKEN')
QUOTES_PATH="quotes.txt"
client = discord.Client()

@client.event
async def on_ready():
    game=False
    ans=""
    # Everything under here, the bot executes for "commands"
    print(f'{client.user} has connected to Discord!')


@client.event
async def on_message(message):
    # client.user = this bot
    # message.author = who sent the message
    # message.content = contents of the message

client.run(TOKEN)

File: javascript starter code/bot.js
const Discord = require('discord.js');
const auth = require('./auth.json');
var mongo = require('mongoose');
const client = new Discord.Client();
const User = require('./model/UserTemplate');

// Initialize MongoDB
mongo.connect(auth.mongo);

mongo.connection.on('connected' , () =>{
    console.log("MongoDB has been connected");
});
mongo.connection.on('err' , (err) =>{
    if (err){
        console.log("MongoDB Error has occured: "+err);
        return
    }
});

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}!\n\n`);
});

client.on("guildCreate", guild => {
    // This event triggers when the bot joins a guild.
    console.log(`New guild joined: ${guild.name} (id: ${guild.id}). This guild has ${guild.memberCount} members!`);
    client.user.setActivity(`Serving ${client.guilds.size} servers`);
  });
  
  client.on("guildDelete", guild => {
    // this event triggers when the bot is removed from a guild.
    console.log(`I have been removed from: ${guild.name} (id: ${guild.id})`);
    client.user.setActivity(`Serving ${client.guilds.size} servers`);
  });



// Juicy stuff goes here

client.on('message', msg => {

    if (msg.author.bot) return;

    // msg.content = msg
    // Tho this sends through the server.

});


client.login(auth.token);